xmodule/seq_block.py

```diff
@@ -978,8 +978,43 @@ def definition_to_xml(self, resource_fs):
         xml_object = etree.Element('sequential')
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
+        self.add_prerequisite_to_xml(xml_object)
         return xml_object
 
+    def add_prerequisite_to_xml(self, xml_object):
```

Comment:
The exports do not contain the information whether a subsection is available as a prerequisite: https://github.com/openedx/edx-platform/blob/d6633243a8397bb56eb21e38cc43efada4f05931/cms/templates/js/access-editor.underscore#L43-L55